src/Thrift/netcore/Thrift/Transports/Client/THttpClientTransport.cs

```diff
@@ -148,7 +150,7 @@ private HttpClient CreateClient()
                 httpClient.Timeout = TimeSpan.FromSeconds(_connectTimeout);
             }
 
-            httpClient.DefaultRequestHeaders.Accept.Add(new MediaTypeWithQualityHeaderValue("application/x-thrift"));
+            httpClient.DefaultRequestHeaders.Accept.Add(XThriftMediaType);
```

Comment:
please elaborate - what is the reason for using a different media type here?

Reply:
That was existing (just pulled out to a variable). If the Accept header should also be updated to the Apache Thrift type then I can do that, too.

Reply:
Looking at the Java client, it doesn't look like it's setting the Accepts header. Is that even necessary at all?

Reply:
Looks like this is how the apache/thrift source this was pulled from is set up
https://github.com/apache/thrift/blob/master/lib/netcore/Thrift/Transports/Client/THttpClientTransport.cs#L151

Reply:
Can you try to set it to ApacheThriftMediaType? This sets the accept header and I assume it works. Can’t test it myself before Monday.